src/kcidb.py

```diff
@@ -0,0 +1,51 @@
+import os
+import sys
+
+import kernelci
+import kernelci.data
+from kernelci.config import load
+from kernelci.cli import Args, Command, parse_opts
+
+
+def _print(msg):
+    print(msg)
+    sys.stdout.flush()
```

Comment:
Please move this as a method in the `cmd_run` class and call it instead of `print()`. It's to get the messages printed directly in the docker-compose logs rather than have to wait for the stdout buffer to flush.